src/environment.ts

```diff
@@ -134,6 +143,7 @@ export class SecureEnvironment implements MembraneBroker {
             const redDescriptor = assign(ObjectCreate(null), blueDescriptor);
             if ('value' in blueDescriptor) {
```

Comment:
`'value' in blueDescriptor ` should be cheap since it's a nulled prototype (but who knows objects and engines may be funny with off the beaten path objects like null prototyped things)